app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -56,8 +56,8 @@ fun PartiesScreen(
 		MVIState.Type.LOADING -> PicoverLoader(modifier = Modifier.fillMaxSize())
 		MVIState.Type.LOADED -> LoadedContent(
 			parties = state.parties,
-			onPartyClick = viewModel::onPartyClick,
-			onFabClick = viewModel::onAddPartyClick,
+			onPartyClick = { viewModel.emitEffect(NavigateToPartyDetails(it)) },
+			onFabClick = { viewModel.emitEffect(NavigateToAddParty) },
```

Comment:
Please check where is the handling of these effects - few lines above, in line 50. We emit some side effect through the `flow`, what involves coroutines and make tests more complex, just to handle it few lines above. What is the benefit of that? Separation? Coverage by unit tests? 
Note that we tested only the "facade", `navContoller.navigate(...)` was not touched, what is the most important part here. So we end up with unit tests of intermediate steps and flow but the most important part is missing. That is really nice lesson, we should be more careful about writing unit tests and think about gained value from them. 
It's not the way that we should test everything because "it's a good practice to good to have tests blah blah". 
Unit test are for preventing from the regression, documenting the code and making the code more stable not to increase the statistics of covered lines.